documentation/modules/configuring/con-config-mirrormaker2-replication.adoc

```diff
@@ -14,33 +15,30 @@ MirrorMaker 2.0 uses:
 
 MirrorMaker 2.0 is based on the Kafka Connect framework, _connectors_ managing the transfer of data between clusters.
 
-MirrorMaker 2.0 uses the following connectors:
+MirrorMaker 2.0 consists of the following connectors:
 
-`MirrorSourceConnector`:: The source connector replicates topics from a source cluster to a target cluster.
+`MirrorSourceConnector`:: The source connector replicates topics from a source cluster to a target cluster. It also replicates ACLs and is necessary for the `MirrorCheckpointConnector` to run.
 `MirrorCheckpointConnector`:: The checkpoint connector periodically tracks offsets. If enabled, it also synchronizes consumer group offsets between the source and target cluster.
 `MirrorHeartbeatConnector`:: The heartbeat connector periodically checks connectivity between the source and target cluster.
 
-The process of _mirroring_ data from one cluster to another cluster is asynchronous.
-The recommended pattern is for messages to be produced locally alongside the source Kafka cluster, then consumed remotely close to the target Kafka cluster.
-
-MirrorMaker 2.0 can be used with more than one source cluster.
+The process of _mirroring_ data from a source cluster to a target cluster is asynchronous.
+Each MirrorMaker 2.0 instance mirrors data from one source cluster to one target cluster. 
+You can use more than one MirrorMaker 2.0 instance to mirror data between any number of clusters.
 
 .Replication across two clusters
 image::mirrormaker.png[MirrorMaker 2.0 replication]
 
 By default, a check for new topics in the source cluster is made every 10 minutes.
 You can change the frequency by adding `refresh.topics.interval.seconds` to the source connector configuration.
-However, increasing the frequency of the operation might affect overall performance.
 
 = Cluster configuration
 
 You can use MirrorMaker 2.0 in _active/passive_ or _active/active_ cluster configurations.
 
-* In an _active/active_ configuration, both clusters are active and provide the same data simultaneously, which is useful if you want to make the same data available locally in different geographical locations.
-* In an _active/passive_ configuration, the data from an active cluster is replicated in a passive cluster, which remains on standby, for example, for data recovery in the event of system failure.
+active/active cluster configuration:: An active/active configuration has two active clusters replicating data bidirectionally. Applications can use either cluster. Each cluster can provide the same data. In this way,  you can make the same data available in different geographical locations.
```

Comment:
The confusing thing for many users here is how the offset configuration (does not) works in active-active clusters. I.e. that consumer offsets for the mirrored topics are nit synchronized back tot he original source cluster. It might be worth adding it here.

Reply:
Added this: _As consumer groups are active in both clusters, consumer offsets for replicated topics are not synchronized back to the source cluster. _